manim/scene/streaming_scene.py

```diff
@@ -0,0 +1,113 @@
+from .. import config
+from ..mobject.frame import FullScreenRectangle as Frame
+from ..renderer.stream_renderer import StreamCairoRenderer
+from ..utils.simple_functions import get_parameters
+from .scene import Scene
+
+
+class Stream:
+    """This class is really intended for inheritance of the style:
+
+    >>> class Streamer(Stream, Scene): # doctest: +SKIP
+    ...     pass
+    ...
+    >>>
+
+    This order is paramount. This :class:`Stream` class does the switch to
+    the specialized renderer, which uses the :class:`StreamFileWriter` to
+    handle specialized streaming services. That explains the calls to super,
+    which dig through the MRO of a class instead of using just a single
+    implementation contained in Scene. Okay, the references in super probably
+    point to things in the :class:`Scene` class only, but it's already happened so...
+
+    PS: You can probably already tell using this class on its own will bring you
+    errors.
+
+    """
+
+    def __init__(self, **kwargs):
+        camera_class = self.mint_camera_class()
+        renderer = StreamCairoRenderer(camera_class=camera_class)
+        super().__init__(renderer=renderer, **kwargs)
+        # To identify the frame in a black background
+        self.add(Frame())
+        # TODO: What happens when setup actually has play arguments?
```

Comment:
**Nitpick** Remove the TODO

Reply:
Okay, I'll remove the TODO and fix the other thing